src/scripts/main.js

```diff
@@ -1,3 +1,20 @@
 'use strict';
 
 // write your code here
+
+const popul = document.querySelectorAll('.population');
+const total = document.querySelector('.total-population');
+const average = document.querySelector('.average-population');
+
+const nums = [...popul].map(people => people.innerHTML.split(',').join(''));
+
+let sum = 0;
+
+for (const num of nums) {
+  sum += +num;
+};
+
+const midl = sum / 10;
```

Comment:
```suggestion
const midl = sum / nums.length;
```
А якщо країн буде більше, ніж 10? 
Не дуже хороша ідея - задавати кількість явно. Ця цифра має бути гнучкою